### ISSUE TITLE
Small accessibility issues on login page

### ISSUE THREAD

**Issue by mtthwhggns** [apluslms/a-plus#629]
- Heading levels don't match
- It would make sense to use `section` for types of login
- The old button contrast issue
- Input ttype should be programmatically determinable - add some `autocomplete` attributes.

**Comment by jaguarfi** [comment]
@Mankro I supposed that the button contrast has already been solved.  What do we exactly want to autocomplete in the login page?

**Comment by mtthwhggns** [comment]
> @Mankro I supposed that the button contrast has already been solved. What do we exactly want to autocomplete in the login page?

Can also help with this, since I guess I wasn't clear enough. 

In this context, `autocomplete` just means the HTML 5.2 attribute. This is related to [WCAG SC 1.3.5](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html) - assistive technology should be able to tell "this is the field for this website's password", or "I should put the username here". It's also potentially used by password managers.

Here's [the relevant technique](https://www.w3.org/WAI/WCAG21/Techniques/html/H98) and [the documentation from Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete). 

Interesting side note; in the draft of the next version of WCAG, there is a new requirement for "Accessible authentication" - some sort of login which doesn't require the user to be able to remember a password. A passing example is "A web site uses a properly marked up email and password as the login authentication." [More on that in the 2.2 draft.](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication.html)

**Comment by jaguarfi** [comment]
@mtthwhggns Thank you so much for the clarification. 👍🏽

**Comment by jaguarfi** [comment]
I was testing A+, and it seems that it is already possible to use autocomplete. I actually have a password manager, and this identifies the user and password input. Chrome also provides suggestions for the username based on values that have been used previously.
As far as I understood, the browsers are able to make use of the autocomplete feature by using the `type` attribute. Of course, in some cases, it is important to be more specific about the `autocomplete` field, e.g., when using forms to collect the first and second name of the user, addresses, postal codes, telephones among others. But, in this case, I think that using only the type works quite well. @mtthwhggns and @Mankro  What is your opinion about this?

**Comment by Mankro** [comment]
Please add the correct `autocomplete` attributes at any rate. That will increase the chances that it works correctly especially with specialized accessibility tools. This is not the same thing as the browser's stored form history. By default, browsers tend to remember any input in text fields. The autocomplete attribute is the standard for declaring what type of information could be automatically filled in.
